File: HotDesk_PRAI/App_Module/Error.ascx.vb

Partial Class App_Module_ErrorReport
    Inherits System.Web.UI.UserControl

    Private _validationGroup As String = ""
    Public Property ValidationGroup() As String
        Get
            Return _validationGroup
        End Get
        Set(ByVal value As String)
            _validationGroup = value
        End Set
    End Property

    Protected Sub Page_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load
        vsSummary.ValidationGroup = Me.ValidationGroup
    End Sub
End Class

File: TMS_HotDesk/App_Module/Error.ascx.cs
public partial class App_Module_ErrorReport : System.Web.UI.UserControl
{
    private string _validationGroup = "";
    public string ValidationGroup
    {
        get { return _validationGroup; }
        set { _validationGroup = value; }
    }

    protected void Page_Load(object sender, System.EventArgs e)
    {
        vsSummary.ValidationGroup = this.ValidationGroup;
    }
}
